apps/api/src/account/account.controller.ts

```diff
@@ -1,29 +1,39 @@
-import { Controller, Get } from '@nestjs/common'
+import { Body, Controller, Get, Put } from '@nestjs/common'
 import { RealmViewSupporters, ViewSupporters } from '@podkrepi-bg/podkrepi-types'
 import { AuthenticatedUser, Public, RoleMatchingMode, Roles } from 'nest-keycloak-connect'
 
 import { KeycloakTokenParsed } from '../auth/keycloak'
+import { UpdatePersonDto } from '../person/dto/update-person.dto'
+import { PersonService } from '../person/person.service'
 import { AccountService } from './account.service'
 
 @Controller('account')
 export class AccountController {
-  constructor(private readonly accountService: AccountService) {}
+  constructor(private readonly accountService: AccountService, private readonly personService: PersonService) {}
 
   @Get('me')
   @Public(false)
-  getMe(
+  async getMe(
     @AuthenticatedUser()
     user: KeycloakTokenParsed,
   ) {
     if (user) {
       // Public authenticated
       console.log(user)
-      return { status: 'authenticated', user }
+      return await this.personService.findOne(user.sub as string);
```

Comment:
Let's wrap the output in a variable so it's like 

```json
{
  user: {
    id: '----'
  }
}
```

instead of 

```json
{
  id:'----'
}
```


```suggestion
      return {
        user: await this.personService.findOne(user.sub as string)
      };
```